packages/components/src/__future__/Tag/RemovableTag/subcomponents/RemoveButton.module.scss

```diff
@@ -45,52 +50,3 @@
     }
   }
 }
-
-.gray {
```

Comment:
If you're going to comment out the props etc. then comment this one too, should either delete it all or comment all 😄 

Also do leave some explanation for commenting out the color, probably in the main `RemovableTag.tsx`